cog/command.py

```diff
@@ -66,15 +61,15 @@ def options(self):
         return dict(self.options_)
 
     def arg_count(self):
-        return self.argc_
+        return len(self.args_)
```

Comment:
After doing some more research I see that Python stores the length of objects to avoid extra work. With that being the case, I think we should not only return `len(self.args_)` here but also get rid of `self.argc_` as a member variable since we only need it for the range statement in `load_args_()`.